tfjs-converter/src/executor/graph_executor.ts

```diff
@@ -45,10 +52,14 @@ export class GraphExecutor implements FunctionExecutor {
   private _functions: {[key: string]: Graph} = {};
   private _functionExecutorMap: {[key: string]: FunctionExecutor} = {};
   private _resourceManager: ResourceManager;
-  private intermediateTensors: NamedTensorsMap = {};
-  private keepIds: Set<number>;
+  // Variables with Async suffix is used for dumping by executeAsync.
+  private tensorsPendingDisposalForAsync: NamedTensorsMap;
+  private idsKeepForAsync: Set<number>;
+  // Variables with Sync suffix is used for dumping by execute.
+  private tensorsPendingDisposalForSync: Tensor[];
+  // tensorsMap is used by both execute and executeAsync.
   private tensorsMap: NamedTensorsMap;
-  private keepTensorForDebug = false;
+  private dumpMode = DumpMode.Default;
```

Comment:
A boolean variable `keepTensorForDebug` like before is enough? I think the `tidy` won't be used for async execute.

Reply:
This is for: 
https://github.com/axinging/tfjs/blob/ca5dbc94cbae0848af0a38d61f58aefe81240c4b/tfjs-converter/src/executor/graph_executor.ts#L346

In Default mode,  it does dispose; Sync mode,  does nothing; Async mode: pushes tensor to disposal queue.
```
                if (this.dumpMode === DumpMode.Default) {
                  tensor.dispose();
                } else if (this.dumpMode === DumpMode.Async) {
                  this.tensorsPendingDisposal.push(tensor);
                }
```

Reply:
In sync mode, it won't go to this code since `tensor.kept = true` since you add `keepTensors` before it?

Reply:
You are right.  Put code under "(this.dumpMode === DumpMode.Async)" help to remind this is dump specific.
BTW, I move this logic outside this checkTensorForDisposal in the latest change.